### ISSUE TITLE
In ?list command show if argument is required.

### ISSUE THREAD

**Issue by likwidoxigen** [jakevoytko/crbot#14]
The list command does not specify which commands require arguments. It makes for having to test unknowns in private message or risk looking foolish.

**Comment by drvan** [comment]
Would changing ?list to return something like the following help?

```
List of builtins:
 - ?help
 - ?learn <call> <response>
 - ?list
 - ?unlearn <call>
```

**Comment by likwidoxigen** [comment]
As long as it tells me something like
?nou <string>
So I know to use
?nou @arc
Instead of getting an error.

On Apr 10, 2017 7:38 PM, "Ryan Van Antwerp" <notifications@github.com>
wrote:

Would changing ?list to return something like the following help?

List of builtins:
 - ?help
 - ?learn <call> <response>
 - ?list
 - ?unlearn <call>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/jakevoytko/crbot/issues/14#issuecomment-293108768>, or mute
the thread

**Comment by drvan** [comment]
Oh - you're not referring to the ?list commands themselves, but rather all learned commands - you'd like to know if '?nou' requires and argument or not. I'll take a look and see what the LoE to implement would be.

**Comment by likwidoxigen** [comment]
Awesome! Thank you!

**Comment by jakevoytko** [comment]
@likwidoxigen do you also want `?help` to list this information for custom commands, or would you only find it useful to have this in `?list`?

**Comment by jakevoytko** [comment]
I'm bored, and as far as I can tell, Bryce has been raptured (or something). Accordingly, I'm making a unilateral decision to append help text to each of the commands in ?list. Additionally, `?help` 
 is changing so that it prints help text for custom commands

**Comment by jakevoytko** [comment]
Fixed by e6f49accf606109fe45f183cc449dc331ae40a30

**Comment by likwidoxigen** [comment]
This sounds like it solves my issue. Thanks!
